recipe/meta.yaml

```diff
@@ -62,7 +62,6 @@ about:
     license: Apache-2.0
     license_family: Apache
     license_file: "{{ name|lower }}-{{ version }}/LICENSE"
```

Comment:
I think LICENSE is in the root of the package. Do we need {{ name|lower }}-{{ version }} part?